### ISSUE TITLE
Multipass transfer truncates large files on Windows

### ISSUE THREAD

**Issue by brudo** [canonical/multipass#3093]
**Describe the bug**
When I try to transfer a large file to a multipass instance on Windows, e.g. a 1.6GB file, the file on the instance is much smaller, e.g. 347 bytes.

**To Reproduce**
How, and what happened?
1. Create or download a large file to the Windows host OS, e.g. the installar tar.gz for Oracle 19C database or another large application
2. `multipass transfer <largefilename>.tar.gz <guest-name>:.`
3. Review the size of the file created on the guest.

**Expected behavior**
The file should be the same size and content as on the Windows host OS, as would occur e.g. if using a mount (although mount is unsecure on Windows).

**Logs**
No log messages appear in Event Viewer from the Multipass source when the multipass transfer command is issued

**Additional info**
 - OS: Windows 10, 22H2, build 19045.2965
- `multipass version`
```
multipass   1.11.1+win
multipassd  1.11.1+win
```
- `multipass info --all`
```
Name:           primary
State:          Running
IPv4:           192.168.125.193
Release:        Ubuntu 22.04.2 LTS
Image hash:     b9a5a216901c (Ubuntu 22.04 LTS)
CPU(s):         2
Load:           0.00 0.00 0.00
Disk usage:     4.6GiB out of 92.9GiB
Memory usage:   232.1MiB out of 5.8GiB
Mounts:         --

Name:           docker
State:          Stopped
IPv4:           --
Release:        --
Image hash:     4401cf7e9948 (Ubuntu 22.04 LTS)
CPU(s):         --
Load:           --
Disk usage:     --
Memory usage:   --
Mounts:         --
``` 
- `multipass get local.driver`
```
hyperv
```

**Additional context**
The same issue does not occur with multipass mount. However, enabling multipass mount is not secure on Windows. For a provisional workaround, we can set local.priviledged-mounts=1, then mount the folder, copy the file, then umount, and set local.priviledged-mounts=0 again.

**Comment by townsend2010** [comment]
Hi @brudo,

I'm pretty sure this is an example of #2990 in which the fix will be released in the upcoming 1.12 release.  I'm going to mark this as a duplicate of that in the meantime.  Thanks!

Duplicate of #2990.

**Comment by brudo** [comment]
OK, sounds like it could very well be related. I scanned for open cases but didn't find that closed one!
